aws-cache-helpers/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@gomomento/poc-aws-cache-helpers",
+  "version": "0.1.0",
```

Comment:
@cprice404 is `0.1.0` right to hard code here?

Reply:
seems fine for now.